src/AsyncMysql/AsyncMysqlConnectResult.php

```diff
@@ -20,7 +20,7 @@ public function __construct(bool $from_pool) {
 
   <<__Override>>
   public function elapsedMicros(): int {
-    return (int)($this->elapsed / 1000000);
+    return (int)($this->elapsed * 1000000);
```

Comment:
Why was this dividing by 1 mil before, was this just a bug? Also is this function actually being used somewhere?

Reply:
I introduced this bug in #40. All usages are in the consumers of this library.